### DISCUSSION TITLE
Allowing poker players to sign up for {name}.bsky.poker

### DISCUSSION THREAD

**Discussion by ChicagoDave** [bluesky-social/atproto#2159] [How to do it?]
I _think_ I understand the atproto and DID specifications but am still struggling with an implementation.

I bought bsky.poker and opened an AWS account with an email address within that domain. My intention is to setup a service to allow BlueSky users to choose a subdomain as their login, specifically under bsky.poker. As an example, I could be david-cornelson.bsky.poker when using my poker identity.

I sense there are DNS requirements (easy) and some API and database requirements (also easy), but I'm not sure about signings and DID documents.

Can someone chime in here?

**Comment by snarfed** [comment]
You wouldn't need to do anything with DIDs, DID docs, keys, or signing. You'd just have users set their bsky.poker handles in their clients, and then serve either DNS or HTTPS to provide handle resolution: https://atproto.com/specs/handle#handle-resolution

**Comment by bnewbold** [comment]
You could take at the `*.bsky.london` setup, they have open source software for running this kind of shared handle namespace:

- https://bsky.london/create-your-own
- https://github.com/mozzius/community-handles
